AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -188,6 +188,11 @@ - (void)setOpaque:(BOOL)opaque
   _placeholderTextKitComponents.textView.opaque = opaque;
 }
 
+- (void)setLayerBacked:(BOOL)layerBacked {
```

Comment:
Open bracket on newline.

Expand the comment below, by revising after the dash - "instances must be view-backed in order to ensure touch events can be passed to the internal UITextView during editing"